package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -210,10 +210,11 @@
 
 import numpy as np
 import MDAnalysis as mda
-from . import base, core
+from . import base
 from ..exceptions import NoDataError
 from ..due import due, Doi
 from MDAnalysis.lib.util import store_init_arguments
+from MDAnalysis.lib.mdamath import triclinic_box
```

Comment:
Can we change these to relative imports like everything else?
```suggestion
from ..lib.util import store_init_arguments
from ..lib.mdamath import triclinic_box
```